packages/deployment/app/cloudformation.template.yml

```diff
@@ -888,13 +964,13 @@ Outputs:
       Ref: {{@key}}SQS
 {{/each}}
 
-{{# if es.name}}
-  {{es.name}}Domain:
-    Value:
-      Fn::GetAtt:
-        - {{es.name}}Domain
-        - DomainEndpoint
-{{/if}}
+# {{# if es.name}}
```

Comment:
why did you comment out this?

Reply:
this is not a duplicate entry. This is an output.

Reply:
oops